pkg/config/authorization.go

```diff
@@ -17,12 +18,12 @@ type AuthorizationConfig struct {
 	JWT *authorization.JWTClaims `yaml:"jwt"`
 }
 
-func (config *AuthorizationConfig) Call(ctx common.AuthContext) (interface{}, error) {
+func (config *AuthorizationConfig) Call(authContext common.AuthContext, ctx context.Context) (interface{}, error) {
```

Comment:
We might want to look for a different name for the `AuthContext` interface and struct... it's not a golang context per se, and it gets weird when one understands that in fact we are passing just one context instead of two.

Reply:
I thought about it as well. I see the confusion and also would like to avoid it. Thing is Golang has appropriated the word "context" which has a way richer meaning in English than what one can do with it in Go. I couldn't find a better name for ours. Maybe "AuthRequest"? (Though it is more than just "request".) Suggestion?

Reply:
Let's put a pin on this, for a future refactor.